README.md

```diff
@@ -1 +1,119 @@
 # egghead-instructor-center
+
+_An app for egghead instructors to do instructor things_
+
+[ ![Codeship Status for eggheadio/egghead-instructor-center](https://app.codeship.com/projects/3a4afe00-8808-0134-e6cc-2e5dfce30ebc/status?branch=master)](https://app.codeship.com/projects/183842)
+
+---
+
+# Getting started
+
+- Ensure you have Git, Node, and Yarn installed
+- Clone this repo and `cd` into it
+- `yarn` to install packages
+- `yarn test` to run tests
+- `yarn dev` to run app
+- Open [localhost:3000](http://localhost:3000) to view app
```

Comment:
@tayiorbeii The `5000` port is if you are running the dev environment locally; the `3000` is for the react app (client), like `2000` in `egghead-ui`